lib/ansible/modules/network/sros/sros_config.py

```diff
@@ -198,6 +199,23 @@
       src: "{{ inventory_hostname }}.cfg"
       provider: "{{ cli }}"
       save: yes
+
+- name: invalid use of lines
+  sros_config:
+    lines:
+      - service
+      -     vpls 1000 customer foo 1 create
+      -         description "invalid lines example"
+    provider: "{{ cli }}"
+
+- name: valid use of lines
+  sros_config:
+    lines:
+      - description "invalid lines example"
```

Comment:
Looks okay... "current context lines" might be misread.

Should say: lines are executed in the context specified. It is not allowed/possible to enter any children context.